config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits.yaml

```diff
@@ -1,3 +1,22 @@
+presets:
```

Comment:
please put this in prow/config.yaml with the other presets, thanks! :pray: 

(it _will_ work here) but others will use this.

Reply:
done